src/disco/gui/fd_gui_printf.c

```diff
@@ -490,7 +490,6 @@ fd_gui_printf_waterfall( fd_gui_t *               gui,
       jsonp_ulong( gui, "net_overrun",       cur->out.net_overrun       - prev->out.net_overrun );
       jsonp_ulong( gui, "quic_overrun",      cur->out.quic_overrun      - prev->out.quic_overrun );
       jsonp_ulong( gui, "quic_frag_drop",    cur->out.quic_frag_drop    - prev->out.quic_frag_drop );
-      jsonp_ulong( gui, "quic_frag_drop_g",  cur->out.quic_frag_drop_g  - prev->out.quic_frag_drop_g );
```

Comment:
Might need to rebuild the frontend, since it expects this

Reply:
Done